cinnabar/remote_helper.py

```diff
@@ -268,51 +269,45 @@ def list(self, arg=None):
                     self._store, new_branchmap, list(new_heads))
 
         refs_style = None
+        refs_styles = ('bookmarks', 'heads', 'tips')
         if not fetch and branchmap.heads():
-            refs_styles = [
-                None,
-                '',
-                'all',
-                'bookmarks',
-                'heads',
-                'tips',
-            ]
-
-            refs_configs = ['cinnabar.refs']
+            refs_config = 'cinnabar.refs'
             if arg == 'for-push':
-                refs_configs.insert(0, 'cinnabar.pushrefs')
+                if Git.config('cinnabar.pushrefs', remote=self._remote.name):
+                    refs_config = 'cinnabar.pushrefs'
 
-            for refs_config in refs_configs:
-                refs_style = Git.config(refs_config, remote=self._remote.name,
-                                        values=refs_styles)
-                if refs_style:
-                    break
+            refs_style = ConfigSetFunc(refs_config, refs_styles,
+                                       remote=self._remote.name, default='all')
 
-        refs_style = refs_style or 'all'
+        refs_style = refs_style or ConfigSetFunc('', refs_styles, default='all')
```

Comment:
flake8 complains about this line:
`cinnabar/remote_helper.py:282:80: E501 line too long (80 > 79 characters)`

That said, I'd rather go with something like `refs_style = refs_style or lambda x: False`

Reply:
Thanks, good suggestion, done